### ISSUE TITLE
[java] Merge IfStmtsMustUseBraces and IfElseStmtsMustUseBraces

### ISSUE THREAD

**Issue by oowekyala** [pmd/pmd#974]
We could use a property to turn off the check for else, though I personally think it should be enabled by default, for consistency.

**Comment by oowekyala** [comment]
Maybe we could even create a new rule "UseBracesOnControlStatement", with properties to control the behaviour for each statement. It would probably be easier for the user to maintain a consistent style across different statements. And it reduces the size of the Code Style category if we deprecate the former rules, which is nice.

**Comment by jsotuyod** [comment]
@oowekyala definitely. I'd have a single rule, maybe even with properties to flag each type in case someone decides to allow no braces around a particular control structure.

**Comment by davidburstromspotify** [comment]
Great work! It seems though as if there is no way to make braces optional for if-statements without else clauses. There is no property for checkIfStatement.

**Comment by davidburstromspotify** [comment]
I see an issue has already been raised in https://github.com/pmd/pmd/issues/1004
